projectile.el

```diff
@@ -2776,6 +2776,142 @@ test/impl/other files as below:
   :type 'function
   :package-version '(projectile . "1.0.0"))
 
+;;;; Constant signifying opting out of CMake preset commands.
+(defconst projectile--cmake-no-preset "*no preset*")
+
+(defun projectile--cmake-version()
```

Comment:
It's idiomatic to have a space between the function name and its param list.

Reply:
Fixed.